app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -106,7 +106,6 @@ def analytics(user: AnonymousUser.new)
   def email_analytics_attributes(enrollment)
     {
       timestamp: Time.zone.now,
-      user_id: enrollment.user_id,
       service_provider: enrollment.issuer,
       wait_until: mail_delivery_params(enrollment.proofed_at)[:wait_until],
     }
```

Comment:
I think that if you add in the `enrollment_code` to this method then this PR would hit the ACs for [LG-9176](https://cm-jira.usa.gov/browse/LG-9176)

Reply:
Oh, my bad, I think we also need to add enrollment_code [here](https://github.com/18F/identity-idp/blob/29fa97d1d18e56593fcd80f8c41dcb91e549f76d/app/jobs/get_usps_proofing_results_job.rb#L280-L283), since expiration emails don't use email_analytics_attributes

Reply:
follow up PR to fix that: #8039